autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>favor string interpolation here and elsewhere</p><pre class="ql-syntax" spellcheck="false">prefix = f'sagemaker/{auto_ml_job_name}'
</pre><p> </p><p>some indentation issues with <code>input_data_config</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
fixed